src/test/java/bankaccount/accounts.java

```diff
@@ -0,0 +1,90 @@
+package bankaccount;
+
+
+
+import java.util.Scanner;
+
+public class accounts {
+
+  int checkingAccountBalance;
+   int savingsAccountBalance;
+   int savingsAccountNum;
+   int checkingAccountNum;
+   private int accountNum = 1000;
+   int numAccount= 0;
+    Scanner scanner= new Scanner(System.in);
+
+    //method to deposit
+    public void deposit(){
+        System.out.println("PLease enter last 4digits account number you would like to deposit from");
+        int last_4= scanner.nextInt();
+        if (last_4==checkingAccountNum || last_4==savingsAccountNum) {
+            System.out.println("Enter the amount you would like to deposit");
+            double amount = scanner.nextDouble();
+            if (amount > 0) {
+                if (last_4 == checkingAccountNum) {
+                    checkingAccountBalance += amount;
+                } else if (last_4 == savingsAccountNum)
+                    checkingAccountBalance += amount;
```

Comment:
I think you intended to write ** savingsAccountBalance** in else if block.

Reply:
Yes that was a mistake